e2e/test/input-tests/input-test.js

```diff
@@ -0,0 +1,50 @@
+const assert = require('assert');
+const visualRegression = require('../../utils/visual-regression');
+const { takeScreenshotAndGetWholePageCompareResult, takeScreenShotOfElement } = visualRegression;
+
+const BASIC_INPUT = '//*[@placeholder="Basic"]';
+
+const BASIC_INPUT_NAME = 'Basic-input';
+const BASIC_INPUT_CLICKED_NAME = 'Basic-input-clicked';
+const BASIC_INPUT_ADDED_TEXT_NAME = 'Basic-input-added-text';
+
+describe('FEF input tests', () => {
+
+  it('Checks the the fef inputs page title and browser compare visual regression', () => {
+    browser.url('/inputs');
+    assert.equal(browser.getTitle(), 'Intellyo Application Design System');
+    assert(takeScreenshotAndGetWholePageCompareResult({defaultTolerance: 14, ignoreComparison: false, testDirPath: __dirname}),
+      'Whole FEF input page screenshot not similar to other configurations');
+  });
+
+  it('should check ' + BASIC_INPUT_NAME, () => {
+    browser.url('/inputs');
+    assert.equal(browser.getTitle(), 'Intellyo Application Design System');
+    const misMatchTolerance = (browser.desiredCapabilities.browserName === 'firefox') ? 11.5 : 4;//TODO : Why firefox problematic?
```

Comment:
you could add `firefoxTolerance` to `takeScreenShotOfElement` as well beyond `windowsTolerance`